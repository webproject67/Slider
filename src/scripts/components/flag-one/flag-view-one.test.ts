import SliderModel from '../model/slider-model';
import FlagViewOne from './flag-view-one';

const model = new SliderModel();
const flag = new FlagViewOne(model);

test('spyOn mouseDown flag one', () => {
  const somethingSpy = jest.spyOn(flag, 'bind').mockImplementation();
  flag.bind();
  expect(somethingSpy).toHaveBeenCalledTimes(1)
});