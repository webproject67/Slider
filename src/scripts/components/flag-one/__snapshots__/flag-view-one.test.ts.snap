// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest snapshots 1`] = `
<div
  class="slider__flags"
>
  
      
  <span
    class="slider__flag slider__flag--max"
    data-testid="flagHorizontal"
    style="left:100%"
  >
    100
  </span>
  
    
</div>
`;
